README.md

```diff
@@ -1,175 +1,115 @@
 # FAImageView
 
-[![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-FAImageView-green.svg?style=flat)](https://android-arsenal.com/details/1/2682)
+FAImageView is a HMOS Library called Frame Animation ImageView. You can animate multiple image.
 
-FAImageView is a Frame Animation ImageView for Android. You can set multiple frame images and start frame animation like UIImageView in iOS. You can animate multiple image like below!
+## Source
 
-![](FAImageView.gif)
+Inspired by [skyfe79/FAImageView](https://github.com/skyfe79/FAImageView) 
 
-## Setup Gradle
+## Features
```

Comment:
Mention about the OhosChannel lib that was used internally for FAImageView lib

Reply:
done